webclient/index.html

```diff
@@ -26,7 +27,7 @@
   <div id="tab-login">
     <h3>Login to server</h3>  
     <label for="host">Host</label>
-    <input type="text" id="host" value="127.0.0.1" />
+    <input type="text" id="host" value="cockatrice.woogerworks.com" />
```

Comment:
Which one is a better default? 127.0.0.1 is probably good only for local testing